buildbots.rst

```diff
@@ -41,7 +43,7 @@ There are three ways of visualizing recent build results:
   to display the latest build results on the development ("master") branch,
   type::
 
-      bbreport.py -q 3.x
+      $ bbreport.py -q 3.x
```

Comment:
'$' is system-specific (why don't you use '>' for the benefit of Windows newbies), just noise, and a nuisance. 
We should be getting rid of them where they are, not adding.

Reply:
As I flagged, I'm taking my lead from `gitbootcamp,rst`. It's not covered in the style guide, except where the text is a Python console and there it is well-established that we *do* show the prompt. Distinguishing input and output seemed useful in some of the console material I visited. Is this an established style rule?

I use Windows and see a `$ ` with my `git` commands because I use git bash. While the prompt where I run Sphinx actually says `devguide PS> `, I'm not phased by seeing `$ ` universally in guides.

Reply:
I usually include the prompt if the output is also present in order to distinguish the two.  If there is no output then I usually omit the prompt to make the commands easy to copy/paste.  For Python snippets in the main docs we solved the problem by adding a button that removes the prompt/output, so the code can be copy/pasted in both cases.

Reply:
The prompt & output stripper is good and I missed it as soon as I came to write my own docs. WIBNI lexers supported it as standard.

Ok, I'll revert chunks of this (most of the added $ signs and the indentation adjustments) and then see what highlight language choice produces the most readable effect. The indenting is non-standard in lots of places, but that's another PR, if at all. Fresh commit to follow.

Reply:
If you notice an highlight style that is prevalent throughout the devguide, we could set that as the default highlight so you don't have to specify it for each code block/document.

Reply:
In case it wasn't clear, I've reverted my insertion of prompts and chosen highlight language correspondingly.